### PR TITLE
fix wrong total audit event number

### PR DESCRIPTION
**Pull request by wanjunlei**
Signed-off-by: wanjunlei wanjunlei@yunify.com

What type of PR is this?
> /kind bug

**Which issue(s) this PR fixes**:
Change total in histogram response to auditing  events count instead of buckets count when search auditing with param operation=histogram

### CODE REVIEW

**Comment by benjaminhuo** [comment]
The code change is good, pls change this PR's title from `fix bug total result of auditing time topology error #2253` to `fix wrong total audit event number` @wanjunlei

File: pkg/models/tenant/tenant.go [hunk 1]
```diff
@@ -702,10 +702,9 @@ func (t *tenantOperator) Auditing(user user.Info, queryParam *auditingv1alpha1.Q
 		listEvts := authorizer.AttributesRecord{
 			User:            user,
 			Verb:            "list",
-			APIGroup:        "",
-			APIVersion:      "v1",
 			Resource:        "namespaces",
 			ResourceRequest: true,
+			ResourceScope:   request.ClusterScope,
```
Comment (benjaminhuo) [inline pkg/models/tenant/tenant.go:707]: No need to add NamespaceScope as events does?

Comment (benjaminhuo) [inline pkg/models/tenant/tenant.go:707]: This looks like a different PR actually

Comment (wanjunlei) [reply pkg/models/tenant/tenant.go]: Delete modification of permission and submit as a pr separately

Comment (wanjunlei) [reply pkg/models/tenant/tenant.go]: Delete modification of permission and submit as a pr separately

**Comment by benjaminhuo** [comment]
@zryfish pls review this PR

**Pull request merged**